### ISSUE TITLE
WSL enable windows vista mode in Adobe PDF reader

### ISSUE THREAD

**Issue by m4ttes4** [microsoft/WSL#11122]
### Windows Version

Microsoft Windows [Version 10.0.22631.3007]

### WSL Version

 2.0.9.0

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.133.1-1

### Distro Version

Ubuntu 22.04

### Other Software

Adobe Acrobat Pdf Reader Version 2023.008.20470

### Repro Steps

after a clean install of Windows 11:
install Adobe
install WSL2 with git
install VS Code

steps:
open Adobe 
open any application that can read data from WSL  and start reading data from WSL (ie: type 'wsl' in terminal, open VS code in remote, navigate in WSL folders from file explorer...)

### Expected Behavior

nothing

### Actual Behavior

Adobe ui changes to Windows Vista-like mode

![Screenshot 2024-01-31 175620](https://github.com/microsoft/WSL/assets/96539845/7119241a-a7fd-4047-9ed0-97306fb3330f)

```[tasklist]
### Tasks
```

**Comment by m4ttes4** [comment]
edit: this issue is related to the compatibility of Adobe, if I change the compatibility of the program adobe.exe (right click, properties, compatibility) I can reproduce the change in UI by enabling the compatibility with older versions of Windows.

**Comment by craigloewen-msft** [comment]
Does this only happen with WSL? Are there other apps that trigger this?
This seems like an issue that is coming from Adobe itself, as I don't think we do anything specific that might cause this.

**Comment by m4ttes4** [comment]
yes, It happens every time an app tries to access data in WSL (terminal, VScode, File Explorer). However, a similar problem occurs when changing the app's compatibility in the properties, enabling compatibility with previous versions of Windows.
